auto-check-fix-commit/README.md

```diff
@@ -17,12 +17,19 @@ Reference - https://git-scm.com/docs/git-bisect/
 
 ### Usage
 
-1. Copy scripts `git-bisect-runner.sh` and `git-bisect-script.sh` to the directory of the project containing the fixed flaky test.
+1. Fork/clone this repo with the auto-check-fix-commit module containing the `git-bisect-runner.sh` and `git-bisect-script.sh` scripts.
 
+2. Within `git-bisect-runner.sh`, modify the `git bisect run ./git-bisect-script.sh $test_module $test_case` line to specify your path to `git-bisect-script.sh`, adding in `nohup` command at the beginning of the command to keep process running even after exiting shell/terminal.
 
-2. Run the script using the below command
+3. Within `git-bisect-script.sh`, modify the build line `mvn clean install -pl $test_module -am -DskipTests -Dlicense.skip` to be the build command for your project with the DeveloperFixed test. Also make sure that the NonDex version, within the script, is correct for your specific project.
+
+4. Within the project containing the DeveloperFixed test, run the script using the command below. Modify arguments to path to script, specific flaky commit, fixed commit, module path, and test case of your project. This command will also ensure standard error and output messages go to `git_bisect_output.log`, while running command in the background.
 ```shell
-./git-bisect-runner.sh --flaky e6d76803f27133d7700811585f5310470e50e487 --fixed 5828d56a824748e7c91076842fad75efb42f92f9 --module core/server/worker --test alluxio.worker.block.BlockLockManagerTest#lockAlreadyReadLockedBlock
```

Comment:
It's good to leave some concrete example in addition to the general invocation.

Reply:
Noted. Added concrete example back in.